hack/common_vars.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+KUBECTL="${REPO_ROOT}/hack/tools/bin/kubectl"
+KIND="${REPO_ROOT}/hack/tools/bin/kind"
+kustomize="${root}/hack/tools/bin/kustomize"
```

Comment:
Should we also capitalize `KUSTOMIZE` so it's consistent?

Reply:
I agree